### ISSUE TITLE
Color blindness emulator

### ISSUE THREAD

**Issue by raunofreiberg** [raunofreiberg/axe-mode#10]
Storybook has an accessibility addon that features a color blindness emulator:

![gif](https://miro.medium.com/max/1400/1*ifHNlVVCz3aSOhULB2NJ1w.gif)

I was thinking if we should maybe have something similar. We would have to add some kind of action menu to switch between different modes or configure them via props.

**Comment by raunofreiberg** [comment]
@chancestrickland wdyt?

**Comment by chancestrickland** [comment]
I do like it, but might it increase the scope of the project a bit for an initial feature? Personally I'd rather nail the core behavior related to axe first as a matter of priorities. Totally see huge value here though!

**Comment by raunofreiberg** [comment]
Yeah. I also feel like this increases the scope too much for the initial release. 

Also, this got me thinking if `axe-mode` should be specifically tied to the `axe` name, or more generally to accessibility instead? Especially with features like this, which do not directly come from `axe`. 

Food for thought?

**Comment by chancestrickland** [comment]
Yeah, it'd be nice to experiment with other tools and see about making them pluggable. Personally I think axe is the best tool of its kind and others will have too many potential false positives, so I don't hate being tied to axe. Deque is a team I trust to keep it solid for the long haul (Disclosure: the lead dev on axe core is a friend of mine here in San Diego, but that's my opinion no less).
